docs/guides/completions.md

```diff
@@ -1,3 +1,83 @@
-Given a prompt, the model will return one or more predicted Completions. The Completions API can be run either
-synchronous or asynchronously (via Python `asyncio`); for each of these modes, you can also choose to stream token
-responses or not.
+LLM Engine provides a list of open source language models (see [Model Zoo](/model_zoo)) that can be used for producing 
```

Comment:
nit: "provides access to various open source..."? feel like the word "list" is unnecessary

Reply:
probably can be worded differently though